packages/fxa-react/styles/ctas.css

```diff
@@ -91,13 +91,13 @@
   }
 
   @screen mobileLandscape {
-    .cta-base {
+    .cta-base:not(.cta-unit-row) {
```

Comment:
If the sole purpose of having this class is so that we can use it in a `:not` here, can we instead create a `.cta-base-common` class or whatever name (shared maybe?), and then extend that for `cta-base`, but directly use the class instead of `cta-unit-row`? It would help us avoid needing this specificity at least and should be a little more maintainable.